internal/pkg/deploy/cloudformation/stack/testdata/workloads/windows-svc-test.params.json

```diff
@@ -1,26 +1,26 @@
 {
-  "Parameters" : { 
+  "Parameters": {
+    "AddonsTemplateURL": "",
     "AppName": "my-app",
-    "EnvName": "test",
-    "WorkloadName": "windows-fe",
     "ContainerImage": "",
-    "AddonsTemplateURL": "",
-    "TaskCPU": "1024",
-    "TaskMemory": "2048",
-    "TaskCount": "1",
-    "LogRetention": "30",
     "ContainerPort": "80",
     "DNSDelegated": "false",
-    "TargetContainer": "windows-fe",
-    "TargetPort": "80",
     "EnvFileARN": "",
-    "RulePath": "/",
+    "EnvName": "test",
     "HTTPSEnabled": "false",
-    "Stickiness": "false"
+    "LogRetention": "30",
```

Comment:
Are we just moving things around for those test files?

Reply:
yeah, since we removed the parameters template and now use `json.Marshal`, the parameters are alphabetized - from [json.Marshal](https://pkg.go.dev/encoding/json#Marshal):
> The map keys are sorted and used as JSON object keys